site/weblogic-docker-images.md

```diff
@@ -64,12 +68,16 @@ Docker Store.
 
 ## Creating a custom image with patches applied
 
+<<<<<<< HEAD
```

Comment:
git merge conflict? There is another one above.

Reply:
I just pushed a merge.  Not quite sure if I did it right.  Not sure if 'master' or 'develop' is supposed to be the lead for the change.   Checking.

Reply:
I just pushed the version that was on develop.

Reply:
So they should match now.